.github/workflows/publishes.yml

```diff
@@ -46,11 +46,11 @@ jobs:
       - name: "Package and Publish-Image-To-DockerHub"
         if: matrix.type == 'image'
         env:
+          IMAGE_NAME: openjdk:8u342
           REGISTRY_USERNAME: ${{ secrets.REGISTRY_USERNAME }}
           REGISTRY_PASSWORD: ${{ secrets.REGISTRY_PASSWORD }}
         run: |
-          echo "IMAGE_NAME=openjdk:8u332" >> $GITHUB_ENV;
-          if [ "${{github.ref}}" == "refs/heads/snapshot" ]; then
+          if [ "${{ github.ref_name }}" == "develop" ] || [ "${{ github.ref_name }}" == "snapshot" ]; then
```

Comment:
why need it ?`"${{ github.ref_name }}" == "develop" `

Reply:
This schedule only takes effect in the default branch `develop`.